src/main/app/AppController.js

```diff
@@ -0,0 +1,108 @@
+const log = require("electron-log");
+
+const ShortcutController = require("./ShortcutController");
+const Updater = require("./../resources/Updater");
+const TrayController = require("./TrayController");
+const MessageHandler = require("./../ipc/MessageHandler");
+const ColorFormats = require("./../resources/ColorFormats");
+
+const HistoryWindowController = require("./../windows/HistoryWindowController");
+const PickerWindowController = require("./../windows/PickerWindowController");
+
+class AppController {
+  constructor(app, store, wm) {
+    // setup logger
+    log.transports.file.level = "silly";

```

Comment:
Have dev mode check here to see how much logging should happen. Maybe have a toggle in the settings so users can provide detailed logs?